include/inviwo/core/ports/datainport.h

```diff
@@ -238,38 +236,41 @@ DataInport<T, N, Flat>::getSourceVectorData() const {
 
 template <typename T, size_t N, bool Flat>
 Document DataInport<T, N, Flat>::getInfo() const {
-    auto name = []() {
-        switch (N) {
-            case 0:
-                return std::string(Flat ? " Flat" : "") + " Multi Inport";
-            case 1:
-                return std::string(Flat ? " Flat" : "") + " Inport";
-            default:
-                return std::string(Flat ? " Flat " : " ") + toString(N) + " Inport";
-        }
-    };
+    StrBuffer name;
+    name.append("{}", DataTraits<T>::dataName());
```

Comment:
```suggestion
    name.append("{}", util::htmlEncode(DataTraits<T>::dataName()));
```
See [dataoutport.h](include/inviwo/core/ports/dataoutport.h)